runtime/lua/vim/treesitter/query.lua

```diff
@@ -1161,10 +1161,23 @@ function M.omnifunc(findstart, base)
   return vim.treesitter._query_linter.omnifunc(findstart, base)
 end
 
+--- Returns available treesitter languages.
+function M._complete()
```

Comment:
oh probably `/runtime/lua/vim/treesitter/language.lua` is a more appropriate place for this